### PR TITLE
fix(docs): minor touch-ups

### PR DESCRIPTION
**Pull request by olaf-k**

### CODE REVIEW

File: libs/components/src/lib/card/README.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ The number of lines are controlled by css variable:
 
 - `--headline-line-clamp`
 - `--subtitle-line-clamp`
-
+<!-- -->
```
Comment (yinonov) [inline libs/components/src/lib/card/README.md:134]: why not simply a line break?

Comment (rachelbt) [reply libs/components/src/lib/card/README.md:134]: it was a requested & designed  by @jshenkman:
https://vonage.slack.com/archives/C013F0YKH99/p1634197624060000

Comment (olaf-k) [reply libs/components/src/lib/card/README.md:134]: > why not simply a line break?
a new line in the middle of a list spaces items evenly
image
to separate two contiguous lists, we could use two `<br>` or one comment, I chose the latter
image

Comment (yinonov) [reply libs/components/src/lib/card/README.md:134]: Each variable should defined separately. And while at it, these are [css syntax](https://developer.mozilla.org/en-US/docs/Web/CSS/@property/syntax) types. Calling it a string do to css pre parsing should be wrong here. These are integers.

If already refactoring, can we align to css syntax?

Comment (olaf-k) [reply libs/components/src/lib/card/README.md:134]: sure

File: libs/components/src/lib/card/README.md [hunk 2]
```diff
@@ -124,29 +124,46 @@ Assign nodes to `main` slot to fully override a card's predefined flow and style
...
+  subtitle="here is the card text"
+>
+</vwc-card>
+```
+### Trim subtitle
+
+The card subtitle can be trimmed to your preferable number of lines.
+The number of lines is controlled by css variable `--subtitle-line-clamp`.
+
+- Type: `number`
```
Comment (yinonov) [inline libs/components/src/lib/card/README.md:154]: it must be a whole number (not fractioned)
```suggestion
- Type: `integer`
```

File: libs/components/src/lib/card/README.md [hunk 3]
```diff
@@ -124,29 +124,46 @@ Assign nodes to `main` slot to fully override a card's predefined flow and style
...
-The card headline and the subtitle can be trimmed to your preferable number of lines.
-The number of lines are controlled by css variable:
+The card headline can be trimmed to your preferable number of lines.
+The number of lines is controlled by the css variable `--headline-line-clamp`.
 
-- `--headline-line-clamp`
-- `--subtitle-line-clamp`
-
-- Type: `string`
+- Type: `number`
```
Comment (yinonov) [inline libs/components/src/lib/card/README.md:132]:
```suggestion
- Type: `integer`
```

File: libs/components/src/lib/layout/README.md [hunk 4]
```diff
@@ -179,8 +179,8 @@ Use the `gutters` attribute to set the initial preferred `margin`.
 Controls the grid-template-columns of the layout.
 
 - `--layout-grid-template-columns`
```
Comment (yinonov) [inline libs/components/src/lib/layout/README.md:181]:
```suggestion
Control the grid-template-columns of the layout by setting `--layout-grid-template-columns`.

```

**Review by yinonov — CHANGES_REQUESTED** [review]

**Review by yinonov — APPROVED** [review]

**Pull request merged**